modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js

```diff
@@ -102,6 +102,7 @@ function EditSXPBlueprintForm({
 	const controllerRef = useRef();
 
 	const [errors, setErrors] = useState([]);
+	const [isTitleEdited, setIsTitleEdited] = useState(false);
```

Comment:
Since this is also checking description, we should rename this to `isTitleAndDescriptionEdited`.